### ISSUE TITLE
ini.options_present should insert an option where it is already present

### ISSUE THREAD

**Issue by arthurlogilab** [saltstack/salt#16958]
For a config file : 

```
[section]
# comment on option1
#option1=
# comment on option2
option2=xxx
```

applying

```
test:
  ini.options_present:
    - sections:
       section:
          option1: myoption
```

results in 

```
[section]
# comment on option1
#option1=
# comment on option2
option2=xxx
option1=myoption
```
it would be nice to have 

```
[section]
# comment on option1
#option1=
option1=myoption
# comment on option2
option2=xxx
```
or

```
[section]
# comment on option1
option1=myoption
# comment on option2
option2=xxx
```

**Comment by Akilesh1597** [comment]
Please check pull request 26359

**Comment by basepi** [comment]
Fixed in #26359
